portfolio/src/main/webapp/index.html

```diff
@@ -34,9 +34,53 @@ <h2>About Me:</h2>
                 shows than I'd ever care to admit, and playing with my puppy, Fitz.
             </p>
         </div>
-        <div id="my-projects" class="tabcontent">
-            <h2>My Projects:</h2>
-            <p>Here are some of my projects</p>
+        <div id="experience" class="tabcontent">
+            <h2>Experience:</h2>
+            <p>Here is a brief snapshot of my work and extracurricular experiences over the years, in a less formal setting than a traditional resume.</p>
+            <div id="resume-item">
```

Comment:
use classes for this not id.
Ids should be unique per page, if you're reusing them, use classes instead

Reply:
Got it! Had the same issue with the travel-pic bunch from yesterday, changed all those to a class as well.